src/KeyboardShortcuts.tsx

```diff
@@ -33,12 +41,20 @@ const bindings: Record<KeyboardAction, string[]> = {
   // NOTE: The "$mod" special value is used to represent the "Control" key on Windows/Linux and the "Command" key on macOS.
   // See https://www.npmjs.com/package/tinykeys#commonly-used-keys-and-codes for more information on the expected values for keyboard shortcuts.
 
-  [KeyboardAction.NEW_QUERY]: ["$mod+J"],
+  [KeyboardAction.NEW_QUERY]: ["$mod+J", "Alt+N Q"],
   [KeyboardAction.EXECUTE_ITEM]: ["Shift+Enter"],
-  [KeyboardAction.CANCEL_QUERY]: ["Escape"],
+  [KeyboardAction.CANCEL_OR_DISCARD]: ["Escape"],
   [KeyboardAction.SAVE_ITEM]: ["$mod+S"],
   [KeyboardAction.OPEN_QUERY]: ["$mod+O"],
   [KeyboardAction.OPEN_QUERY_FROM_DISK]: ["$mod+Shift+O"],
+  [KeyboardAction.NEW_SPROC]: ["Alt+N P"],
+  [KeyboardAction.NEW_UDF]: ["Alt+N F"],
+  [KeyboardAction.NEW_TRIGGER]: ["Alt+N T"],
+  [KeyboardAction.NEW_DATABASE]: ["Alt+N D"],
+  [KeyboardAction.NEW_COLLECTION]: ["Alt+N C"],
+  [KeyboardAction.NEW_ITEM]: ["Ctrl+I", "Alt+N I"],
```

Comment:
🤦🏻‍♀️ Yep, it definitely should be. And even when testing on Windows, `Ctrl` doesn't match the Ctrl key, you have to put `Control` in. I'll fix this up!

Reply:
The other factor with this is that `Ctrl+I` conflicts with opening autocomplete in the Monaco editor. So that keyboard shortcut won't work when focus is in the Item editor itself. The `Alt+N I` shortcut _does_ work when in the editor. Given that, I think I'm actually going to remove the `Ctrl+I` binding entirely for now, so there's a consistent shortcut that works in all contexts.